### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by SteffenDE**
Work in progress

### CODE REVIEW

**Comment by saleyn** [comment]
I made it a little easier to see the test match reasons in CI:
https://github.com/saleyn/erlexec/runs/7321194951?check_suite_focus=true

**Comment by SteffenDE** [comment]
Yeah my change is probably not needed then; I see the `Cmd given as a unicode string` part failing and one in the winsz test. Looking at the winsz test now.

**Review by saleyn — APPROVED** [review]

**Review by saleyn — APPROVED** [review]

**Comment by saleyn** [comment]
If `tput` is still failing, maybe try `tput -T xterm lines cols`?

File: src/exec.erl [hunk 1]
```diff
@@ -1424,78 +1424,41 @@ print(Stream, OsPid, Data) ->
...
-                               test     => TestName,
-                               line     => Line})
+receiveBytes(Fd, Pid, Bytes, Timeout) ->
+    (fun Recv(N, Acc) ->
+        receive
+            {Fd, Pid, Data} -> case byte_size(Data) of
+            M when M >= N ->
+                <<Want:N/binary, Rest/binary>> = Data,
+                % maybe this is a bad idea, as the order might be wrong
+                self() ! Rest,
```
[the remark this answers is not in the document]
Comment (SteffenDE) [reply src/exec.erl:1453]: Yeah, as the comment already says.. Either one has to return the rest and pass it to the function again, or one would need some kind of IO process that simply blocks until the specific number of bytes arrived and buffers the rest.

**Comment by saleyn** [comment]
Thank you very much!

**Comment by SteffenDE** [comment]
Sorry for the back and forth; your `check_receive` function combined with the PORT_DEBUG log confused me, therefore I unnecessarily moved back to my `receiveBytes` function. One might see that I'm coming from the Elixir world and my Erlang is rather bad, but in the end I moved back to your solution as it works fine.

The only real changes are the macOS pipeline fix, as well as the assertion for the long "test\r\ntest\r\n". Thanks for the patience 😄

**Pull request merged**